File: gyozelem/client-app/src/components/event-calendar/CalendarEvent.ts
import { ICalendar } from "./types";

export class CalendarEvent implements ICalendar.Event {
    public id: number;
    public user_id?: number;
    public title: string;
    public message: string;
    public createdAt?: string;
    public updatedAt?: string;
}

export default CalendarEvent;

File: gyozelem/Domain/CalendarEvent.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace Domain
{
    public class CalendarEvent
    {
        public Guid Id { get; set; }
        public DateTime StartAt { get; set; }
        public DateTime? EndAt { get; set; }
        public uint VisibilityLevel { get; set; }
        public string Title { get; set; }
        public string Message { get; set; }
        public DateTime CreatedAt { get; set; }
        public Guid CreatedBy { get; set; }
        public DateTime? UpdatedAt { get; set; }
        public Guid? UpdatedBy { get; set; }
        public virtual List<CalendarGuest> CalendarGuests { get; set; }
    }
}
